src/main/java/com/minecolonies/coremod/client/gui/WindowCitizen.java

```diff
@@ -260,6 +260,16 @@ private static void addHeart(final View healthBarView, final int heartPos, final
         healthBarView.addChild(heartImage);
     }
 
+    private static int getYOffset(final int i)
```

Comment:
Please add javadoc to those two methods. On a quick look they're not obvious. I understood them because I know what the code is supposed to do in the first place.